Scripts/Editor/Configuration/WitConfigurationEditor.cs

```diff
@@ -136,7 +144,14 @@ public override void OnInspectorGUI()
 
         if (hasApplicationInfo)
         {
-            selectedToolPanel = GUILayout.Toolbar(selectedToolPanel, toolPanelNames);
+            if (configuration.application.id != null && !configuration.application.id.StartsWith("voice"))
```

Comment:
I tried just about every variant of is empty + int.TryParse(), etc. and this was the combination I could get to reliably perform the way we want it to. Other scenarios ran into problems with the app id not being fetched at the time that we display this info.